src/components/language-picker.tsx

```diff
@@ -16,6 +16,8 @@ export const LanguagePicker = () => {
 					value: locale as SupportedLocale,
 					label,
 				} ) ) }
+				__nextHasNoMarginBottom={ true }
```

Comment:
Nitpick: we could skip the `true` here.

Reply:
I removed it in https://github.com/Automattic/studio/pull/742/commits/c9a9f8fb1983d096ab8710796cac3a0624657058. As a sidenote, we are passing it explicitly in a couple of other places within the app.